README.md

```diff
@@ -1,3 +1,4 @@
 # BT3103
 
 NUS Business Analytics course
+This line will cause conflict
```

Comment:
Can you keep the README intact?

Reply:
Hi sorry I just realised the merge wasnt registered but I tried to resolve it on github now. 

Is that okay now? I resolved the conflict of the readme by removing the extra line